Userland/Libraries/LibIPC/File.h

```diff
@@ -41,8 +41,8 @@ class File {
     }
 
     template<typename... Args>
```

Comment:
The `template` should be removed (and we can make this constructor `explicit` while we are here)